templates/web/pages/import_product_categories_process/import_product_categories_from_public_database_process.tt.html

```diff
@@ -0,0 +1,10 @@
+<!-- start templates/[% template.name %] -->
+
+<p> job_id: [% job_id %]</p>
+<a href="[% link %]">status</a>
+
+</br>Poll: <span id="poll"></span> 
```

Comment:
I don't think </br> is valid, it can be simply <br>, or <br/> if we want valid XHTML (but that's not needed).